readme.md

```diff
@@ -88,6 +88,7 @@ save your subscription.
     - **tokenUrl** *string*: where the service should get the accesstoken
     - **redirectUri (optional)** *string*: custom redirect callback uri, do not use unless absolutely necessary
     - **refresh** *boolean*: whether or not to refresh the access token (experimental)
+    - **refreshMax** *number*: In seconds, if set, will refresh the session until it reached its max age based on this setting (Only if refresh is set to true)
```

Comment:
Voor mij was het niet meteen duidelijk wat die refreshMax wou zeggen. Is iets als "maxAgeRefreshToken" geen betere naam?

Reply:
Agreed

Reply:
Nu ik er wat meer over heb nagedacht is deze naam ook niet helemaal correct, het gaat eigenlijk over de maxAge van het userToken (dat in tussentijd wel refreshed is geweest) :smile: